services/catalog/src/simcore_service_catalog/core/background_tasks.py

```diff
@@ -232,3 +234,11 @@ async def stop_registry_sync_task(app: FastAPI) -> None:
             await task
         app.state.registry_sync_task = None
     _logger.info("registry syncing task stopped")
+
+
+async def setup_background_task(app: FastAPI) -> AsyncIterator[State]:
+    await start_registry_sync_task(app)
+    try:
+        yield {}
+    finally:
```

Comment:
for later: this should use the service lib background task

Reply:
Yes. I will definitveily follow up on that one